### PR TITLE
Add iopipe helper module example

### PR DESCRIPTION
**Pull request by coreylight**
The helper module allows a user to use methods like `.mark` and `.log` throughout the application, not just in the "main" module.

### CODE REVIEW

File: node/helper-module/index.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  // do your work as normal
+  const pongo = await typeOne();
+  const perdita = await typeTwo();
+
+  // add IOpipe labels to the invocation, 1 label per dog breed
+  const breeds = [pongo, perdita].map(url => url.match(/\/breeds\/(.*)\//)[1]);
+  breeds.forEach(label);
+
+  // add IOpipe custom metrics for each url
+  log('pongo-url', pongo);
```
Comment (kolanos) [inline node/helper-module/index.js]: @coreylight Don't we want to encourage use of `metric()` since `log()` is being deprecated?

Comment (coreylight) [reply node/helper-module/index.js]: @kolanos yup you're right, I'll change that. Thanks!

**Comment by coreylight** [comment]
I can point to the invocations in the IOpipe dashboard for review.

File: node/helper-module/serverless.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+service: example-tracing-as-module-can-delete
```
Comment (pselle) [inline node/helper-module/serverless.yml]: `example-tracing-as-module-can-delete`? perhaps something referencing the helper module?

File: node/helper-module/package.json [hunk 3]
```diff
@@ -0,0 +1,19 @@
+{
+  "name": "tracing-as-module",
```
Comment (pselle) [inline node/helper-module/package.json]: Name update here too for consistency?

Comment (coreylight) [reply node/helper-module/package.json]: You got it. Updated

**Pull request merged**